### PR TITLE
fix: summary panel nrql query issues (Fixes #10)

### LINKED ISSUE #10: summary-panel renders inconsistently
The summary-panel renders inconsistently, when switching between processes.  To be more specific, the panel renders on initial load, but then may not render at all when switching between processes

## Description

The summary-panel (top-right, which includes the commandName, commandLine and parent/child process id's) will occasionally not render when selecting a new process, or after re-selecting a process which had been previously selected and for which the panel previously rendered.

## Steps to Reproduce

1) In NR1, locate a host, and load the Top nerdlet.  Select a process other than the one currently selected at load, re-select the original process. The summary panel does not render.

2) In NR1, locate a host, and load the Top nerdlet.  Select a process other than the one currently selected at load, then select another.  After several selections, the summary panel no longer renders.  Note, navigating using the child/parent link yields the same result.

## Expected Behaviour

The summary-panel should load consistently, for every process

## Your Environment

* Versions 0.1.5, 0.1.6
* Browsers: 
Chrome Version 77.0.3865.120 (Official Build) (64-bit) (Mac)
Firefox 69.0.3 (64-bit) (Mac)

### PR DESCRIPTION
**Pull request by jbeveland27**
- fixes summary panel disappearing act by replacing the `NrqlQuery` component with `NerdGraphQuery` (also utilizes `NerdGraphError` for convenient error handling)
    - Additional Info: Repeated fetches with the same Nrql query using the `NrqlQuery` component fail to return data when `NrqlQuery.FORMAT_TYPE.RAW` is specified. I did a test with the SINCE clause continuously updating vs remaining static to determine this is an issue. It seems something in the service itself is not responding with data, so we may need to look into this more in case it impacts other Nerdpacks. Switching `NrqlQuery` out for `NerdGraphQuery` addressed the issue. 
- Tidying up styling so summary/charts don't jump as much when different processes are selected. Also fixed some spacing issues by updating Stack props.
- Updated Icon.SIZE_TYPE to comply with warning from dev tools console.
- Added nr1.json to root of project

### CODE REVIEW

File: nerdlets/top-nerdlet/process-details.js [hunk 1]
```diff
@@ -77,19 +98,48 @@ export default class ProcessDetails extends React.PureComponent {
...
     return (
-      <NrqlQuery accountId={entity.accountId} query={nrql} formatType="raw">
-        {({ loading, data }) => {
-          if (loading) return null;
-          const { results } = data;
-          if (!results || results.length == 0) return null;
+      <NerdGraphQuery query={nrql}>
+        {({ loading, error, data }) => {
+          if (loading) {
+            return <Spinner style={{ height: '110px' }} />;
```
Comment (tangollama) [inline nerdlets/top-nerdlet/process-details.js]: What about just fillContainer vs. a custom style?

Comment (jbeveland27) [reply nerdlets/top-nerdlet/process-details.js]: We can do that. The difference is fillContainer covers the entire right panel (summary + charts) whereas this height restriction made it so the spinner covered just the summary component (i.e. less overall "flashing" when a new process is selected). I'm good to change it to fillContainer if we want to favor using the built-in props.

**Comment by jbeveland27** [comment]
Looks like all the changes are present in the files now. Not sure why the 4th commit didn't come through in the conversation but ¯\_(ツ)_/¯

**Pull request merged**